src/ServicePulse.Host/vue/src/components/failedmessages/MessageView.vue

```diff
@@ -364,29 +403,29 @@ onUnmounted(() => {
           <div class="row">
             <div class="col-sm-12 no-side-padding">
               <div class="active break group-title">
-                <h1 class="message-type-title">{{ failedMessage.message_type }}</h1>
+                <h1 class="message-type-title">{{ failedMessage?.message_type }}</h1>
               </div>
             </div>
           </div>
           <div class="row">
             <div class="col-sm-12 no-side-padding">
               <div class="metadata group-message-count message-metadata">
-                <span v-if="failedMessage.retried" title="Message is being retried" class="label sidebar-label label-info metadata-label">Retried</span>
-                <span v-if="failedMessage.restoring" title="Message is being retried" class="label sidebar-label label-info metadata-label">Restoring...</span>
-                <span v-if="failedMessage.archiving" title="Message is being deleted" class="label sidebar-label label-info metadata-label">Deleting...</span>
-                <span v-if="failedMessage.archived" title="Message is being deleted" class="label sidebar-label label-warning metadata-label">Deleted</span>
-                <span v-if="failedMessage.resolved" title="Message was processed successfully" class="label sidebar-label label-warning metadata-label">Processed</span>
-                <span v-if="failedMessage.number_of_processing_attempts > 1" :title="'This message has already failed ' + failedMessage.number_of_processing_attempts + ' times'" class="label sidebar-label label-important metadata-label"
-                  >{{ failedMessage.number_of_processing_attempts }} Retry Failures</span
+                <span v-if="failedMessage?.retried" title="Message is being retried" class="label sidebar-label label-info metadata-label">Retried</span>
+                <span v-if="failedMessage?.restoring" title="Message is being retried" class="label sidebar-label label-info metadata-label">Restoring...</span>
+                <span v-if="failedMessage?.archiving" title="Message is being deleted" class="label sidebar-label label-info metadata-label">Deleting...</span>
+                <span v-if="failedMessage?.archived" title="Message is being deleted" class="label sidebar-label label-warning metadata-label">Deleted</span>
+                <span v-if="failedMessage?.resolved" title="Message was processed successfully" class="label sidebar-label label-warning metadata-label">Processed</span>
+                <span v-if="failedMessage?.number_of_processing_attempts ?? 0 > 1" :title="'This message has already failed ' + failedMessage?.number_of_processing_attempts + ' times'" class="label sidebar-label label-important metadata-label"
+                  >{{ failedMessage?.number_of_processing_attempts }} Retry Failures</span
```

Comment:
I'm not sure the logic is correct here (nothing to do with this PR). The retry failures would be the number_of_processing_attempts - 1, since the first processing attempt wouldn't be a retry